Document N_0x45f35c0edc33b03b (#909)
citizenfx/natives: NETWORK/NetworkAddMapObjectToSynchronisedScene.md

---
ns: NETWORK
aliases: ["0x45F35C0EDC33B03B"]
---
## _NETWORK_ADD_MAP_OBJECT_TO_SYNCHRONISED_SCENE

```c
// 0x45F35C0EDC33B03B

> void _NETWORK_ADD_MAP_OBJECT_TO_SYNCHRONISED_SCENE(int netScene, Hash modelHash, float x, float y, float z, cs_type(float) char* animDict, char* animName, float p7, float p8, int flags);

Reviewer: Looking at the parameter ordering, which is similar to other methods such as [NETWORK_ADD_PED_TO_SYNCHRONISED_SCENE](https://github.com/citizenfx/natives/blob/master/NETWORK/NetworkAddPedToSynchronisedScene.md) and [TASK_PLAY_ANIM](https://github.com/citizenfx/natives/blob/master/TASK/TaskPlayAnim.md) `p7` may be `blendInSpeed` and `p8` may be `blendOutSpeed`.

The decompiled method structures (where these parameters get applied) for both `NETWORK_ADD_PED_TO_SYNCHRONISED_SCENE` and this are also similar.

Author: thank you, i've added them

```

> Adds a map object/entity to a synchronised scene, by specifying it's hash.

Reviewer 2: ```suggestion
Adds a map object/entity to a synchronised scene.
```

Reviewer 2: Keep it short & concise

## Parameters

> * **netScene**: net scene id, returned by NetworkCreateSynchronisedScene
> * **modelHash**: Object model hash

Reviewer 2: ```suggestion
* **netScene**: The net scene's ID, which is returned by NetworkCreateSynchronisedScene
* **modelHash**: The object model's hash
```

> * **x**: x coord
> * **y**: y coord
> * **z**: z coord

Reviewer 2: ```suggestion
* **x**: X coordinate
* **y**: Y coordinate
* **z**: Z coordinate
```

Reviewer 2: change coord to coordinate, since that is the correct spelling
some people might read "coords" instead of "coord" at a first glance

> * **animDict**: anim dictionary to play on this object
> * **animName**: anim name to play on this object

Reviewer 2: ```suggestion
* **animDict**: The animation dictionary to play on this object
* **animName**: The animation name to play on this object
```

> * **p7**:
> * **p8**:

Reviewer 2: ```suggestion
* **p7**: unknown, although gta seems to use `2.0`
* **p8**: unknown, although gta seems to use `-1.5`
```

Shouldn't `p7 and `p8` either be explained in the Parameters list.
Or if its unknown, just note the default value, no?
